api/index.ts

```diff
@@ -4,18 +4,25 @@ import { MongoClient } from 'mongodb';
 import getDBConnection from '../db';
 
 export default async (req: NowRequest, res: NowResponse) => {
-  const db = await getDBConnection();
-  db.collection("docs").findOne({ name: req.body.name }, (err, result) => {
-    if (err || result === null) {
-      res.send({
-        success: false,
-        err
-      });
+    const db = await getDBConnection();
+    const cursor = await db.collection("docs").find();
+    if (await cursor.count() == 0) {
+        res.send({
+            success: false
+        })
     } else {
-      res.send({
-        success: true,
-        data: result
-      });
+        // Re-format json data
+        let data = await cursor.toArray();
+        let docs = {};
+        for (let val of data) {
+            docs[val.name] = {
+                link: val.link,
+                docstring: val.docstring
+            }
```

Comment:
semicolon

Reply:
Also you could do this in the following way if you wanted
```
const { name, link, docstring } = val;
docs[name] = { link, docstring };
```
or even
```
for (const { name, link, docstring } of data) {
  docs[name] = { link, docstring };
}
```
No need to change unless you want to. Just a fun fact.